pkg/costmodel/cluster.go

```diff
@@ -44,6 +45,11 @@ const (
 
 const MAX_LOCAL_STORAGE_SIZE = 1024 * 1024 * 1024 * 1024
 
+// When ASSET_INCLUDE_LOCAL_DISK_COST is set to false, local storage
+// provisioned by sig-storage-local-static-provisioner is excluded
+// by checking if the volume is prefixed by "local-pv-"
+const SIG_STORAGE_LOCAL_PROVISIONER_PREFIX = "local-pv-"
```

Comment:
Would it be worthwhile adding this code ref in the comments here? https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/b6f465027bd059e92c0032c81dd1e1d90e35c909/pkg/discovery/discovery.go#L410-L417

Reply:
Done!